src/libraries/System.Console/src/System/Console.cs

```diff
@@ -33,16 +33,28 @@ public static class Console
         private static ConsoleCancelEventHandler? s_cancelCallbacks;
         private static ConsolePal.ControlCHandlerRegistrar? s_registrar;
 
-        internal static T EnsureInitialized<T>([NotNull] ref T? field, Func<T> initializer) where T : class =>
-            LazyInitializer.EnsureInitialized(ref field, ref InternalSyncObject, initializer);
+        internal static T EnsureInitializedDisposable<T>([AllowNull] ref T? field, Func<T> initializer) where T : class, IDisposable
```

Comment:
Why did this change from [NotNull] to [AllowNull]?  The type being `T?` means that null is allowed in; the `[NotNull]` was saying it won't be null on the way out.

Reply:
It was not intentional, bad copy paste...